### PR TITLE
Tweaks weed node damage threshold

### LINKED ISSUE #2375: Weed nodes should take 2 hits to break, not 1

### PR DESCRIPTION
**Pull request by vero5123**
## About the PR
closes #2375 
## Why / Balance
requested tweak, for reference it takes two hits with a bayonet knife to destroy nodes now.

## Technical details
increased the threshold from 5 to 30 to destroy nodes; takes 2 hits with a standard bayonet knife and more with entities that have lower damage output. Similar to cm13.

## Media
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
N/A

**Changelog**
:cl:
- tweak: increased the required damage threshold to destroy weed nodes

### CODE REVIEW

**Comment by DrSmugleaf** [comment]
I need to ask Skarlet about this but from testing in CMSS13 the weed nodes do take one hit to destroy with a bayonet for me, not two

File: Resources/Prototypes/_RMC14/Entities/Structures/Xeno/xeno_weeds.yml [hunk 1]
```diff
@@ -109,6 +109,14 @@
     layers:
     - state: weed0
     - state: constructionnode
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 30
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Xeno/xeno_weeds.yml:116]:
```suggestion
        damage: 33
```

File: Resources/Prototypes/_RMC14/Entities/Structures/Xeno/xeno_weeds.yml [hunk 2]
```diff
@@ -182,6 +190,14 @@
     layers:
     - state: hive_weed0
     - state: constructionnode
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 30
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Xeno/xeno_weeds.yml:197]:
```suggestion
        damage: 33
```

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
smug says it should be 33

**Review by QuietlyWhisper — APPROVED** [review]
thanks

**Pull request merged**